app/views/devise/shared/_links.html.erb

```diff
@@ -1,13 +1,8 @@
-<% confirmation = devise_mapping.confirmable? && ['confirmations', 'registrations', 'sessions'].include?(controller_name) %>
```

Comment:
I now this has been requested, but it's not a good idea. we get requests from encore to resend confirmations fairly regularly, because there isn't a resend confirmation link on their site. people sign up, don't click the confirm link in the email, then come back months later when they've deleted the email, so at that point they point they have to contact site admins, who contact us, and we have to log onto the server to send the email manually

Reply:
Agree - have reverted for now, and will flag to Charlotte